uui/components/pickers/__tests__/PickerInput.test.tsx

```diff
@@ -121,51 +161,921 @@ describe('PickerInput', () => {
         );
         expect(tree).toMatchSnapshot();
     });
+    
+    describe('[selectionMode single]', () => {
+        it('should select & clear option', async () => {
+            const { dom, mocks } = await setupPickerInputForTest({
+                value: undefined,
+                selectionMode: 'single',
+            });
+            expect(dom.input?.getAttribute('placeholder')?.trim()).toEqual('Please select');
```

Comment:
please avoid any DOM specific API as much as possible. In this case it's better to check it as: ```expect(screen.queryByPlaceholderText('Please select', { exact: false })).toBeInTheDocument();```

Reply:
I want to query placeholder of the exact input, not of some at the component. Why should I query over all the page (even if all the page is PickerInput component)? What if there will be inputs at the rendered rows? This example is highlighting all the problems of such an approach. It is very unstable.